content/post/daily-bugle-boot2-root-writeup copy.md

```diff
@@ -0,0 +1,734 @@
+---
+title: "Daily Bugle Boot2Root Write-Up"
+date: 2020-11-27T00:00:00-04:00
+author: "Nate Catelli"
+tags: ["ctf", "boot2root", "hacking", "writeup", "tryhackme"]
+description: "A boot2root writeup of the Daily Bugle host from TryHackMe"
+type: "post"
+draft: false
+---
+
+## Introduction:
+The Daily Bugle challenge was exceptionally difficult compared to a few of the other hosts I've attacked in the past. Not specifically for any technical reason but it provided many opportunities to rabbithole if I wasn't dilligent in both thinking about how I looked for information but also in what I documented. I will attempt to call out the points that I really rabbitholed on but I think it will be very difficult to express in written form.
```

Comment:
... compared to challenges that I've attempted in the past.

Reply:
It was incredibly easy to rabbithole down some path if I wasn't diligent both in thinking about how to collect information but also in what information I documented.

Reply:
In this post, I'll talk through my experience in tackling this challenge and especially focus on the rabbitholes I fell victim to, however it may be difficult to express that in writing.